recipes/editor/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
This file is in the tarball. There is no need to re-add it here. It will likely get out of sync if the tarball changes.

Reply:
Looks like all other licenses are also in the tarball. Please remove them all.